myosuite/tests/test_envs.py

```diff
@@ -25,57 +37,87 @@ def check_envs(self, module_name, env_names, lite=False, input_seed=1234):
 
 
     def check_env(self, environment_id, input_seed):
+
+        # If requested, skip tests for envs that requires encoder downloading
+        ROBOHIVE_TEST = os.getenv('ROBOHIVE_TEST')
```

Comment:
We  do not have this environmental variable, do we?

Reply:
That is for anyone who wants to define it. Default behavior is "not defined".